deploy/crds/argoproj.io_argocds_crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
apiextensions.k8s.io/v1 does not support fields like validation, it should be replaced with schema field.